### ISSUE TITLE
Visualise shape disputes & represent shape uncertainty

### ISSUE THREAD

**Issue by edwardchalstrey1** [edwardchalstrey1/seshat#46]
Visualise shape disputes:
- After #45 (perhaps before) we may have disputes over shape extents included in the db as alternative shapes. Find a suitable way to visualise these.
- Territorial disputes and conflicts

**Comment by edwardchalstrey1** [comment]
Represent uncertainty:
- Can we have some kind of varying confidence score
- Could be visually represented with shading, fuzzy borders

**Comment by edwardchalstrey1** [comment]
Jim Bennet has an idea for this - chat to him

**Comment by edwardchalstrey1** [comment]
Closing as superseded by https://github.com/Seshat-Global-History-Databank/seshat/issues/20
